azure/packages/external-controller/webpack.config.js

```diff
@@ -18,6 +18,9 @@ module.exports = (env) => {
             },
             resolve: {
                 extensions: [".ts", ".tsx", ".js"],
+                fallback: {
+                    assert: require.resolve("assert/"),
+                },
```

Comment:
This is because we're still on older server bits in azure? Either way out of scope to deal with this in this PR.